CMakeLists.txt

```diff
@@ -212,7 +212,7 @@ elseif (("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU") OR ("${CMAKE_C_COMPILER_ID}" S
 	set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3")
 
 	# enable somewhat modern C++
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
```

Comment:
A strictly conforming ISO C math.h wouldn't have `_isnan()`, which is a Microsoft extension. If you specify strict ISO conformance, then you don't get that function declared and the build fails.

I could try making `Q_isnan()` use `isnan()` on mingw-w64 (by changing the conditional for `_isnan()` from `_WIN32` to `_MSC_VER`), but `-std=gnu++11` is the closest C++11 equivalent of how OpenJK was compiled before #738.